Makefile

```diff
@@ -61,6 +64,9 @@ $(CPP_OBJECTS): $(CPP_SRC) $(CPP_HEADERS)
 $(BIN): $(CPY) $(MAIN_SRC) $(OBJECTS) $(CPP_OBJECTS)
 	$(COBC) -x $(COBC_OPTS) -I $(CPY_DIR) -lstdc++ -lz -o $@ $(MAIN_SRC) $(OBJECTS) $(CPP_OBJECTS)
 
-test: $(CPY) $(TEST_SRC) $(OBJECTS) $(CPP_OBJECTS)
-	$(COBC) -x $(COBC_OPTS) -I $(CPY_DIR) -lstdc++ -lz -o $@ $(TEST_SRC) $(OBJECTS) $(CPP_OBJECTS)
+$(TEST_OBJECTS): tests/%.o: tests/%.cob $(CPY)
```

Comment:
I would prefer if objects weren't placed next to sources. Besides looks, a practical reason is that `make clean` will only delete objects for which source files are still present, but if source files are renamed or removed, the objects will stay. With a separate directory, we can just delete the directory.

Perhaps place the test objects into a directory `test-out`? This would need to be added to `.dockerignore` and `.gitignore` as well, then.

Reply:
sounds good, I'll put it in out/test/ (the former directory is needed in any case)